users/migrations/0008_merge_20191117_2007.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.6 on 2019-11-17 20:07
```

Comment:
(Python3 generated from `python3 manage.py makemigrations --merge` as instructed in the terminal)